library/src/main/java/me/mattlogan/library/ParcelableIntStack.java

```diff
@@ -0,0 +1,52 @@
+package me.mattlogan.library;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import java.util.Stack;
+
+/**
+ * It's a normal Stack of ints.  Only Parcelable!
+ */
+public class ParcelableIntStack extends Stack<Integer> implements Parcelable {
+
+    public ParcelableIntStack() {
```

Comment:
Is this default constructor necessary? Sorry I'm not super familiar with Parcelable.